app/parse-siti-payment-file/index.js

```diff
@@ -0,0 +1,52 @@
+const fs = require('fs')
+const readline = require('readline')
+
+const transformBatch = require('./transform-batch')
+const transformHeaders = require('./transform-headers')
+const transformLines = require('./transform-lines')
+const invoiceToPaymentRequest = require('./payment-request')
+const validate = require('./validate')
+
+const invoiceBatch = []
+const invoiceHeaders = []
+
+const parseInvoiceLineType = (invoiceLine) => {
+  const invoiceLineType = invoiceLine[0]
+
+  switch (invoiceLineType) {
+    case 'B':
+      invoiceBatch.push(transformBatch(invoiceLine))
+      break
+    case 'H':
+      invoiceHeaders.push(transformHeaders(invoiceLine))
+      break
+    case 'L':
+      invoiceHeaders[invoiceHeaders.length - 1]
+        .lines
+        .push(transformLines(invoiceLine))
+      break
+  }
+}
+
+const parseFile = (fileBuffer) => {
+  const invoiceInput = fs.createReadStream(fileBuffer)
```

Comment:
This doesn't appear to work with a FileBuffer, `fs.createReadStream` only accepts a file name.

Had a quick hack (read StackOverflow) and this seems to be what we want: 

```
const { Readable } = require('stream')
const invoiceInput = Readable.from(fileBuffer)
```